handler/src/main/java/io/netty/handler/ssl/SslContext.java

```diff
@@ -1089,15 +1090,16 @@ private static PrivateKey getPrivateKeyFromByteBuffer(ByteBuf encodedKeyBuf, Str
      * Build a {@link TrustManagerFactory} from a certificate chain file.
      * @param certChainFile The certificate file to build from.
      * @param trustManagerFactory The existing {@link TrustManagerFactory} that will be used if not {@code null}.
+     * @param keyType The KeyStore Type you want to use
      * @return A {@link TrustManagerFactory} which contains the certificates in {@code certChainFile}
      */
     @Deprecated
     protected static TrustManagerFactory buildTrustManagerFactory(
-            File certChainFile, TrustManagerFactory trustManagerFactory)
+            File certChainFile, TrustManagerFactory trustManagerFactory, String keyType)
```

Comment:
we will need to keep the old method signature as well as this method is protected and so changing its signature is a API breaking change